examples/basic.html

```diff
@@ -39,12 +62,25 @@ <h2><dfn>Foo</dfn> interface</h2>
       <p>The <a>Foo</a> interface is nice. Lets you do stuff.</p>
       <p>The <dfn>bar</dfn> attribute, returns 🍺.</p>
       <p>The <dfn>doTheFoo()</dfn> method, returns nothing.</p>
+      <p>Normative reference [[!ECMASCRIPT]].</p>
```

Comment:
Ah, seems we forgot to document that `[[!` is obsolete.  I updated the Wiki. 

ReSpec now intelligently works out if a reference is normative or informative based on the context of the reference. So, if the reference is inside a note or a figure, then it's treated as non-normative.